learningpath-api/src/main/scala/no/ndla/learningpathapi/service/UpdateService.scala

```diff
@@ -810,6 +832,7 @@ trait UpdateService {
     ): Try[Unit] = {
       implicit val session: DBSession = folderRepository.getSession(readOnly = false)
       val feideId                     = getUserFeideID(feideAccessToken).?
+      val _                           = canWriteDuringWriteRestrictionsOrAccessDenied(feideId, feideAccessToken).?
```

Comment:
Denne trenger vel ikke assignes til noe?

```suggestion
      canWriteDuringWriteRestrictionsOrAccessDenied(feideId, feideAccessToken).?
```